src/Features/CSharp/Portable/Copilot/CSharpCopilotCodeFixProvider.DocumentChangeCodeAction.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Immutable;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+
+namespace Microsoft.CodeAnalysis.CSharp.Copilot
+{
+    internal sealed partial class CSharpCopilotCodeFixProvider
+    {
+        private sealed class CopilotDocumentChangeCodeAction : CodeAction.DocumentChangeAction
+        {
+            internal sealed override ImmutableArray<CodeAction> AdditionalPreviewFlavors { get; }
```

Comment:
this pr is lackign in information and images to know what these types correspond to in terms of ux.

it's unclear why these are additional flavors, and we can't just use the normal flavor codepaths.